application/actions/organization_test.go

```diff
@@ -4,11 +4,13 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
```

Comment:
I like to keep the internal packages grouped at the top.

Reply:
that must have been done automatically by GoLand